Ex3/Dictionary.java

```diff
@@ -0,0 +1,24 @@
+import java.util.TreeSet;
+
+public class Dictionary {
```

Comment:
I might choose a different name for this class, just because there is already a class java.util.Dictionary, and having another one with the same name makes the code a bit harder to read.